pkg/kv/kvtest/store.go

```diff
@@ -146,14 +163,26 @@ func testStoreSetGet(t *testing.T, ms MakeStore) {
 	t.Run("get_missing", func(t *testing.T) {
 		// get a missing key
 		keyNotExists := uniqueKey("key-not-exists")
-		val3, err := store.Get(ctx, keyNotExists)
+		val3, err := store.Get(ctx, []byte(testPartitionKey), keyNotExists)
 		if !errors.Is(err, kv.ErrNotFound) {
 			t.Fatalf("get key='%s' err=%s, expected not found", keyNotExists, err)
 		}
 		if val3 != nil {
 			t.Fatalf("get key='%s' value='%s', expected nil", keyNotExists, val3)
 		}
 	})
+
+	t.Run("get_missing_partition_key", func(t *testing.T) {
+		// get a missing key
+		partitionKeyNotExists := uniqueKey("partition-key-not-exists")
+		val3, err := store.Get(ctx, partitionKeyNotExists, testKey)
+		if !errors.Is(err, kv.ErrNotFound) {
+			t.Fatalf("get key='%s' err=%s, expected not found", partitionKeyNotExists, err)
+		}
+		if val3 != nil {
+			t.Fatalf("get key='%s' value='%s', expected nil", partitionKeyNotExists, val3)
+		}
+	})
```

Comment:
How about adding a key with a (valid and exist) partition key and trying to get it with another (also valid and exist) partition key?

Reply:
Yep, I think that covers it.